Formula/asdf.rb

```diff
@@ -4,29 +4,17 @@ class Asdf < Formula
   url "https://github.com/asdf-vm/asdf/archive/v0.8.0.tar.gz"
   sha256 "9b667ca135c194f38d823c62cc0dc3dbe00d7a9f60caa0c06ecb3047944eadfa"
   license "MIT"
-  revision 1
+  revision 2
   head "https://github.com/asdf-vm/asdf.git"
 
   bottle :unneeded
 
-  depends_on "autoconf"
-  depends_on "automake"
-  depends_on "coreutils"
-  depends_on "libtool"
-  depends_on "libyaml"
-  depends_on "openssl@1.1"
-  depends_on "readline"
-  depends_on "unixodbc"
-
-  conflicts_with "homeshick",
-    because: "asdf and homeshick both install files in lib/commands"
```

Comment:
The lib directory no longer needs to be publicly available?

Reply:
@SMillerDev You see, that's the problem. The current installation would `prefix.install` everything which would just create symlinks the files in folders such as `bin` and `lib` - that being said, I am not 100% sure why someone did that, there could be a reason for it. 

My guess again, is because the way sourcing scripts through relative paths worked. But it is completely unnecessary. 
Keep everything in a single directory, and just symlink the correct entry-point, which I have guessed is `bin/asdf` but I am open to suggestion to what other file should be the correct executable.